### PR TITLE
Expose other advice methods in FirmSerializer

### PR DESCRIPTION
**Pull request by esbel**
* expose names of other advice methods in FirmSearch to allow elastic search index on them

part of #5725

### CODE REVIEW

**Comment by lshepstone** [comment]
Perhaps it'd be better to see this as part of the bigger picture for #5725?

**Comment by esbel** [comment]
Agree. However, that's probably the only change to rad-core required to complete #5725 (an optimist here!).

**Comment by lshepstone** [comment]
Aah, right! So, we need to have a chat about if we're indexing names or IDs for reference data.

**Comment by esbel** [comment]
@lshepstone @benlovell `name`s changed to `id`s, as discussed

**Comment by lshepstone** [comment]
LGTM :+1: Might need a rebase now, though! ;)

**Comment by esbel** [comment]
yes, it did - done.

File: app/serializers/firm_serializer.rb [hunk 1]
```diff
@@ -34,4 +35,8 @@ def wills_and_probate
   def _id
     object.id
   end
+
+  def other_advice_methods
+    object.other_advice_methods.collect(&:id)
```
Comment (benlovell) [inline app/serializers/firm_serializer.rb]: `object.other_advice_method_ids` is more efficient.

Comment (esbel) [inline app/serializers/firm_serializer.rb]: changed.

**Comment by benlovell** [comment]
:ship:

**Comment by esbel** [comment]
cheers @lshepstone @benlovell

**Pull request merged**